tests/Lykke.AlgoStore.Tests/Unit/CodeBuildTests.cs

```diff
@@ -0,0 +1,182 @@
+            var code = Encoding.UTF8.GetString(Convert.FromBase64String(base64Content));
+            var session = GetCSharpCodeBuildSession(code);
+
+            var validationResult = session.Validate().Result;
+
+            Assert.AreEqual(true, validationResult.IsSuccessful);
+
+            var metadata = session.ExtractMetadata().Result;
+            var result = JsonConvert.SerializeObject(metadata);
```

Comment:
We're not testing the JSON serializer, so consider manually testing the returned structure for validity instead